### ISSUE TITLE
Search links contain truncated title when title contains '

### ISSUE THREAD

**Issue by ilg** [rhollister/goodreads#50]
When looking at "The Liars' Club" by Mary Karr, the search results look correct, but the links are to search for `The Liars` at each given library (see `href` of link at lower right).
<img width="748" alt="Screenshot 2023-11-18 at 2 21 10 PM" src="https://github.com/rhollister/goodreads/assets/228228/f0afcb9a-dc2e-42ff-b625-71210fc962d5">

**Comment by p1nkyy** [comment]
Indeed, issue still persists. If you can kindly fix this @rhollister

**Comment by auspex** [comment]
Is this an issue of not properly encoding the URL? So it terminates at the apostrophe?

**Comment by p1nkyy** [comment]
> Is this an issue of not properly encoding the URL? So it terminates at the apostrophe?

yes it seems so.

**Comment by p1nkyy** [comment]
@hminiken Apologies, I don't know how to open an issue in a forked project but can you pls solve this bug in your fork? I'm currently using yours and it seems like it's working well. Thank you in advance!
